<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>314</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>321</width>
      <height>351</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Calculater</string>
     </attribute>
     <widget class="QLabel" name="lblOutput">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>241</width>
        <height>20</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LayoutDirection::RightToLeft</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtNum">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>241</width>
        <height>20</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::FocusPolicy::ClickFocus</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Button1">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Button2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>80</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Button4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>130</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Button5">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>130</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Button6">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>130</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAdd">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>80</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSubtract">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>80</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnDevide">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>130</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>/</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnMultiplay">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>130</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>x</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Button9">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>180</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Button8">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>180</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Button7">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>180</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Button0">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>230</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnBreak">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>230</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>.</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnCE">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>230</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>CE</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnC">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>180</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>C</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnEquel">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>180</y>
        <width>41</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Button3">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>80</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="History">
     <attribute name="title">
      <string>History</string>
     </attribute>
     <widget class="QListView" name="listView">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>256</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
